Parsers/ASimDns/ARM/ASimDns/README.md

```diff
@@ -1,15 +1,15 @@
-# Infoblox NIOS ASIM DNS Normalization Parser
+# Source agnostic ASIM Dns Normalization Parser
 
-This template deploys the ASIM DNS schema parser for Infoblox NIOS. The parser is a part of the Azure Sentinel Information Model.
+This template deploys the ASIM Source agnostic Dns schema parser. The parser is a part of the Azure Sentinel Information Model.
 
 The Azure Sentinel Information Mode (ASIM) enables you to use and create source-agnostic content, simplifying your analysis of the data in your Azure Sentinel workspace.
 
 For more information, see:
 
 - [Normalization and the Azure Sentinel Information Model (ASIM)](https://aka.ms/AzSentinelNormalization)
-- [Azure Sentinel DNS normalization schema reference](https://aka.ms/AzSentinelDnsDoc)
+- [Azure Sentinel Dns normalization schema reference](https://aka.ms/AzSentinelDnsDoc)
```

Comment:
Capitalizing DNS seems like a recurring issue.

Reply:
Yes, I will change texts, but prefer not to rename the function names. Also, DnsEvents and DnsInventory are capitalized the same.